jahia-test-module/src/react/server/views/testUrl/TestUrl.tsx

```diff
@@ -126,7 +126,7 @@ jahiaComponent(
           </a>
         </div>
         <div data-testid="no_weakref">
-          <a href={buildUrl({ path: null })}>No weakref</a>
+          <a href={buildNodeUrl({ nodePath: undefined })}>No weakref</a>
```

Comment:
I don't like this test because it shows that we deal silently with errors. We'll update it later

Reply:
I would say this test had sense before TypeScript was in place. Now it should just be a compilation error...

Reply:
100%